openapi.yaml

```
@@ -840,6 +864,9 @@ components:
               enum:
                 - BEFORE
     cq-external:
+      description: "Allows uploading of external files. Primarily as a list of included entities, with additional options to specify durations or auxiliary virtual selects.
```

```suggestion
      description: "Allows uploading of externally defined cohort, with additional data. Primarily as a list of included entities, with additional options to specify durations or auxiliary virtual selects.
```

Files würde ich weglassen, weil die api es schon strukturiert erwarten und keinen blob entgegen nimmt.